homeassistant/components/binary_sensor/owlet.py

```diff
@@ -0,0 +1,85 @@
+"""
+Support for Owlet binary sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.owlet/
+"""
+from datetime import timedelta
+
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.owlet import DOMAIN
+from homeassistant.util import dt as dt_util
+
+SCAN_INTERVAL = timedelta(seconds=15)
```

Comment:
I'm worried about the amount of traffic caused by refreshing from the cloud every 15 seconds 24/7. The HA use cases for this data I'm imagining only need this data every few minutes at the most.

Reply:
I updated SCAN_INTERVAL to 120 seconds.  That should strike a decent balance between traffic and responsiveness.